src/Sylius/Bundle/ApiBundle/Swagger/PathHiderDocumentationNormalizer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Swagger;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+class PathHiderDocumentationNormalizer implements NormalizerInterface
```

Comment:
you should add spec for this class

Reply:
We don't have specs for any DocumentationNormalizer in our codebase 🦖